### PR TITLE
Setup GH Action to annotate Pull Requests with Larastan Code Analysis

### PR DESCRIPTION
**Pull request by tcampbPPU**
In this PR we could prob combine `lint-php.yml` and the new `larastan.yml` workflow files but idk what do you think? Plus side of having its own way of identifying which action it came from like below:

> image

Also added a `.nvmrc` so the proper node js gets used each time when opening for example iterated terminal in VS Code

References issue #62

### CODE REVIEW

**Comment by dammy001** [comment]
Yeah this makes sense @tcampbPPU

**Comment by dammy001** [comment]
@acidjazz what's your take on this?

**Comment by dammy001** [comment]
Alright that's fine. @tcampbPPU

**Comment by acidjazz** [comment]
> @acidjazz what's your take on this?

agreed! separate workflows help let us determine where/when this runs to optimize our valuable GA minutes

**Comment by tcampbPPU** [comment]
Increased to Level 8 and got all warnings fixed
image

File: app/Exceptions/Handler.php [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Handler extends ExceptionHandler
     /**
      * A list of the exception types that are not reported.
      *
-     * @var array
+     * @var array|string[]
```
Comment (dammy001) [inline app/Exceptions/Handler.php:12]: this can be `array<int, class-string<Throwable>>`

File: app/Exceptions/Handler.php [hunk 2]
```diff
@@ -18,7 +18,7 @@ class Handler extends ExceptionHandler
     /**
      * A list of the inputs that are never flashed for validation exceptions.
      *
-     * @var array
+     * @var array|string[]
```
Comment (dammy001) [inline app/Exceptions/Handler.php:21]: this can be `array<int, string>`

File: .github/workflows/larastan.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  stan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.0'
```
Comment (dammy001) [inline .github/workflows/larastan.yml:20]: I think you should add 8.1 too

Comment (tcampbPPU) [reply .github/workflows/larastan.yml]: Ok I can set that up

**Pull request merged**